modules/appeals_api/spec/fixtures/valid_200996_v2_extra.json

```diff
@@ -17,6 +24,7 @@
           "email": "josie@example.com",
           "timezone": "America/Chicago"
         },
+        "ssoSsocOptIn": true,
```

Comment:
This is the wrong key. Should be `socOptIn`

Reply:
On it